src/FSharp.Data.GraphQL.Client.DesignTime/ProvidedTypesHelper.fs

```diff
@@ -313,23 +270,16 @@ module internal ProvidedOperation =
                     <@@ let rec mapper (value : obj) =
```

Comment:
Could we give `mapper` a more descriptive name?

Reply:
Done.